sources/CMakeLists.txt

```
@@ -17,7 +17,8 @@ add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/neko_ext)
 target_sources(neko-top
     PUBLIC
         ${CMAKE_CURRENT_SOURCE_DIR}/neko_top.f90
-
+	${CMAKE_CURRENT_SOURCE_DIR}/optimizer.f90
+	
```

Please do not mix formatting of files. The file use 4 spaces for indentation
levels. This list of files are indented 2 levels.
You added a new line with tabulated indentation and a single level, which does
not match the existing code.
```suggestion
        ${CMAKE_CURRENT_SOURCE_DIR}/optimizer.f90

```